qml/main.qml

```diff
@@ -112,6 +112,35 @@ ApplicationWindow {
         application.quitOnLastWindowClosed = !settings.closeToTray
     }
 
+    Shortcut {
+        sequence: StandardKey.Copy
+        enabled: currentCredentialCard !== undefined && currentCredentialCard
+        onActivated: app.currentCredentialCard.calculateCard(true)
+    }
+
+    Shortcut {
+        sequence: StandardKey.Delete
+        enabled: currentCredentialCard !== undefined && currentCredentialCard
```

Comment:
You can shorten this to `!!currentCredentialCard` to check for both null and undefined.